### ISSUE TITLE
Trigger Lighthouse webhooks on tag push events

### ISSUE THREAD

**Issue by souvik-deepsource** [jenkins-x/lighthouse#1327]
Currently, Lighthouse does not support tag push event webhooks from GitHub. It would be great if options could be provided to the `postsubmit` config to allow various types of events to trigger pipelines on.

**Comment by dippynark** [comment]
Not sure if this is what you mean, but you can trigger on tag creation using something like this: https://github.com/dippynark/kfmt/blob/master/.lighthouse/triggers.yaml#L13

**Comment by souvik-deepsource** [comment]
@dippynark Can `branches` in `postsubmits` include regexes for tags? I haven't seen any sort of documentation regarding this.

**Comment by dippynark** [comment]
@souvik-deepsource yeah exactly, so that linked line means you can push a local semantic tag and it'll trigger the postsubmit job with PULL_BASE_REF set to that tag. I haven't docs for this either though.

However, like you describe, being able to trigger other types of Git events (e.g. issue comments) would be a useful addition, which I don't think is supported atm. In particular, the trigger plugin checks that the comment is coming from a PR afaik, but it would be cool to trigger actions on issue comments too.

**Comment by jstrachan** [comment]
agreed with all that. Also would be nice to have a simpler syntax to process arbitrary webhook events https://github.com/jenkins-x/lighthouse/issues/1222
